core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -302,21 +316,24 @@ <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-head
           </a>
           <img ng-src="<[getStaticImageUrl('/general/mobile_path_segment.svg')]>">
           <div style="position: fixed; z-index: 100; top: 100px; left: 50%; transform: translate(-130px);">
-            <exploration-summary-tile ng-if="explorationCardIsShown"
-                                      collection-id="getCollectionId()"
-                                      exploration-id="currentExplorationId"
-                                      exploration-title="summaryToPreview.title"
-                                      last-updated-msec="summaryToPreview.last_updated_msec"
-                                      objective="summaryToPreview.objective"
-                                      category="summaryToPreview.category"
-                                      ratings="summaryToPreview.ratings"
-                                      num-views="summaryToPreview.num_views"
-                                      thumbnail-icon-url="summaryToPreview.thumbnail_icon_url"
-                                      thumbnail-bg-color="summaryToPreview.thumbnail_bg_color"
-                                      is-community-owned="summaryToPreview.community_owned"
-                                      is-collection-preview-tile="true"
-                                      style="position: absolute; z-index: 10;">
-            </exploration-summary-tile>
+            <div ng-class="{'oppia-activity-summary-tile-mobile-background-mask': explorationCardIsShown}" ng-click="closeOnClickingOutside($event)" >
```

Comment:
Remove space before the closing ">". Remove the unneeded $event argument.

Reply:
Done